config/environments/development.rb

```diff
@@ -37,6 +37,9 @@
 
   config.action_mailer.perform_caching = false
 
+  # Set default mailer host to be the same as Rails default host to generate urls for mailers.
+  config.action_mailer.default_url_options = { host: 'example.com' }
```

Comment:
In `config/environments/test.rb` the value for this is `{ host: 'localhost', port: 3000 }`. Should we perhaps make this the same for consistency sake?

Reply:
I agree it would be better to have consistency with the test env, I thought it would only configure the host for action_mailer and it wouldn't be consistent in the rest of the app, but I didn't find any evidence of that